src/test/java/core/basesyntax/service/impl/DataConverterImplTest.java

```diff
@@ -0,0 +1,82 @@
+package core.basesyntax.service.impl;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.DataConverter;
+import java.util.List;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+class DataConverterImplTest {
+    private static DataConverter dataConverter;
+
+    @BeforeAll
+    public static void setUp() {
+        dataConverter = new DataConverterImpl();
+    }
+
+    @Test
+    public void numOfElementsIs3_Of() {

```

Comment:
test names should have such pattern `methodName_parameters_behavior`. Fix it everywhere 
```suggestion
    public void convertToTransaction_validData_ok() {
```